packages/orbit-components/src/Box/__tests__/index.test.tsx

```diff
@@ -74,33 +102,55 @@ describe("#Box", () => {
       convertHexToRgba(hex, NaN).replace(", NaN", "").replace("rgba", "rgb");
 
     expect(el).toBeInTheDocument();
-    expect(el).toHaveStyle({ color: getOmittedHex(theme.orbit.paletteBlueDark) });
-    expect(el).toHaveStyle({ background: getOmittedHex(theme.orbit.paletteCloudLight) });
+    expect(el.tagName).toBe("DIV");
+    expect(el).toHaveStyle({ "box-sizing": "border-box" });
+    expect(el).toHaveStyle({ "font-family": theme.orbit.fontFamily });
+    expect(el).toHaveStyle({
+      color: expect.stringContaining(getOmittedHex(theme.orbit.paletteBlueDark)),
```

Comment:
I have just noticed that this test always passes, regardless of the token provided as argument. So will have to figure how to test this.

Reply:
It has to be something with the toHaveStyle matcher. What you can try -> is to manually add logs inside the node_modules folder to that matcher and see how it behaves. I tried a bit and it seems that it simply can't resolve the color 🤔 So either the matcher or our custom render function. I will try to look more at that

Reply:
I have removed the test until we find a way to assert it